Neos.Utility.ObjectHandling/Classes/TypeHandling.php

```diff
@@ -51,7 +51,9 @@ public static function parseType(string $type): array
             throw new InvalidTypeException('Found an invalid element type declaration. A type "' . var_export($type, true) . '" does not exist.', 1264093630);
         }
 
-        $type = self::normalizeType($matches['type']);
+        $typeWithoutNull = self::stripNullableType($matches['type']);
+        $isNullable = $typeWithoutNull !== $matches['type'];
```

Comment:
Couldn't spot a usage for this variable within the function

Reply:
Indeed, that was an attempt to already make the method return nullablity info for the type... but then that would be changing behaviour with possible side-effects in a bugfix which is bad.